1-exercises/B-while-loop/exercise.js

```diff
@@ -6,9 +6,16 @@
 */
 
 function evenNumbers(n) {
-    // TODO
+    let i = 0;
+    let evens = [];
+    while (n > evens.length) {
+        evens[0] = 0;
+        i += 2;
+        evens.push(i);
+    }
+    return evens;
```

Comment:
Can you please review the question that they asked? I think you should use console.log instead of return and store the result in a string, not in an array.

Reply:
You are right. Thanks for catching that